pkg/kubernetes/logs.go

```diff
@@ -45,16 +58,7 @@ func Logs(appID, podName, namespace string) error {
 			for _, container := range pod.Spec.Containers {
 				if container.Name == daprdContainerName {
 					// find app ID
-					for i, arg := range container.Args {
-						if arg == appIDContainerArgName {
-							id := container.Args[i+1]
-							if id == appID {
-								podName = pod.Name
-								foundDaprPod = true
-								break
-							}
-						}
-					}
+					findappid(container, appID, pod, podName, foundDaprPod)
```

Comment:
@tanvigour Isn't this pass by value for foundDaprPod? Does it not break line 61 ? And if so why is not part of E2E tests?